resources/calico/calico.yaml

```diff
@@ -73,11 +46,47 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            # Calico network backend
+            - name: CALICO_NETWORKING_BACKEND
```

Comment:
Health checking strategy still depends on which backend is in use so this isn't really configurable. Its mostly harmless, we'll assume upstream has plans.